app/models/patient.rb

```diff
@@ -53,7 +52,9 @@ class Patient < ApplicationRecord
      monitored_address_state
      preferred_contact_method
      preferred_contact_time
-     sex].each do |enum_field|
+     sex
+     primary_telephone_type
+     secondary_telephone_type].each do |enum_field|
     validates enum_field, on: :api, inclusion: {
       in: VALID_ENUMS[enum_field],
       message: "is not an acceptable value, acceptable values are: '#{VALID_ENUMS[enum_field].join("', '")}'"
```

Comment:
Could you add tests to the Patient model for this validation and error messages? Same with the monitoring plan above.

Reply:
Forgot to respond but I did this here: https://github.com/SaraAlert/SaraAlert/pull/521/commits/7036c8f9e31602a13f26ca2375261a2359dfc9e3